### PR TITLE
fix: Promo Video updates

### PR DESCRIPTION
**Pull request by AndyEPhipps**
#### What is it doing?
Some additions and simplification to make the new functionality better fit for purpose

#### Why is this required?
Good content = nice

#### link to Jira ticket:
No ticket, just an extension of the original rly

### Quick Checklist:
- [x] My PR title follows the Conventional Commit spec.

- [x] I have filled out the PR description as per the template above.

- [ ] ~~I have added tests to cover new or changed behaviour.~~ to follow in a separate PR

- [x] I have updated any relevant documentation.

### CODE REVIEW

**Comment by AndyEPhipps** [comment]
Showing how pre-existing non-video Promos haven't been effected:

https://deploy-preview-594--cr-component-library.netlify.app/#!/Promo/1
https://cr-component-library.netlify.app/#!/Promo/1

**Comment by AndyEPhipps** [comment]
Sorry Mike, found a few issues; will reassign for review once it's actual sorted :/

File: src/components/Molecules/Promo/Promo.md [hunk 1]
```diff
@@ -173,12 +169,12 @@ let mobileVideoSrc = 'https://samplelib.com/lib/preview/mp4/sample-10s.mp4';
   imageLow={defaultData.promoImage}
   imageAltText=""
   copyLeft={true}
-  hasOverlay={true}
   position="upper"
   videoSrc={videoSrc}
   poster={poster}
   loop={true}
   autoplay={true}
+  behindTextGradient='black'
```
Comment (AndyEPhipps) [inline src/components/Molecules/Promo/Promo.md:177]: While these examples are set up to show loads of different combinations of the various settings, within CRcom, there'll be some logic to handle these; 

i.e, when copy is set to 'white', it'll programmatically assign a black gradient overlay and a black play button.

**Comment by AndyEPhipps** [comment]
@KrupaPammi, any idea what's happening with Playwright here? I wasn't aware we had any PW testing in Component Library, plus I can't actually see any PW processes running under [Actions](https://github.com/comicrelief/component-library/actions) so this just seems to be blocking my work :/

**Comment by KrupaPammi** [comment]
Hi @AndyEPhipps, it's part of this PR [here](https://github.com/comicrelief/component-library/pull/592) that I had to park to look into FSU and Donate tests failing issues. I had to add playwright workflow to run on Github actions but will disable it now. So, the change of cypress to playwright is due to intermittent cypress test failures and some outdated tests. What I thought was to add playwright tests which are fast, and add only functionality-related tests instead of every single component, as they are already covered in Jest tests. Please let me know if this is of any concern.

**Comment by KrupaPammi** [comment]
@AndyEPhipps I have now removed the PW workflow from GitHub Actions.

**Comment by AndyEPhipps** [comment]
Thanks @KrupaPammi, but do let me know if this just makes your life more difficult in terms of running and confirming the tests you're developing actually outside of your local?

**Comment by KrupaPammi** [comment]
> > Hi @AndyEPhipps, it's part of this PR [here](https://github.com/comicrelief/component-library/pull/592) that I had to park to look into FSU and Donate tests failing issues. I had to add playwright workflow to run on Github actions but will disable it now. So, the change of cypress to playwright is due to intermittent cypress test failures and some outdated tests. What I thought was to add playwright tests which are fast, and add only functionality-related tests instead of every single component, as they are already covered in Jest tests. Please let me know if this is of any concern.
> 
> Ah right okay, thank you for the clarification; that all sounds sensible to me.
> 
> What do you think would be the timeline for that work, do you have much more to do in that PR? Would it just be possible to remove the branch protection rule _for now_ without it disrupting your work?
> 
> I guess in the worst case, we just have to force-merge these PRs while you're working on the PW stuff.. I'm not sure if that'll stop the NPM Publish job happening here or not, but I'll give it a go now!

@AndyEPhipps I can work on those PW tests locally, as I now know the setup on GitHub actions is working fine so keeping the PW workflow on branch rules disabled should be fine until the tests are ready to be reviewed and merged. By this, none of the PRs or merging will be affected.

**Comment by AndyEPhipps** [comment]
Okay brilliant, thank you!

**Pull request merged**